src/lib/api/container/client/containerApiClient.client.test.ts

```diff
@@ -94,6 +96,35 @@ describe('containerApiClient', () => {
         expect(result.unwrapError()).toBe(mockError.message);
       });
     });
+
+    describe('deleteContainer', () => {
+      it('should return Ok result on success', async () => {
+        /* Arrange */
+        (request as jest.Mock).mockResolvedValue({});
+
+        /* Act */
+        const result = await deleteContainer(mockContainerId);
+
+        /* Assert */
+        expect(result.unwrap()).toBe(undefined);
+        expect(request).toHaveBeenCalledWith({
+          url: expect.stringContaining(`/containers/${mockContainerId}`),
+          method: 'DELETE',
+        });
+      });
+
+      it('should return Err result if API request fails', async () => {
+        /* Arrange */
+        const mockError = new Error('API error');
+        (request as jest.Mock).mockRejectedValue(mockError);
+
+        /* Act */
+        const result = await deleteContainer(mockContainerId);
+
+        /* Assert */
+        expect(result.unwrapError()).toBe(mockError.message);
```

Comment:
Adding `expect(result.err).toBeTruthy();` will improve the clarity of this assertion.

```ts
expect(result.err).toBeTruthy();
expect(result.unwrapError()).toBe(mockError.message);
```